src/pyOpenMS/pxds/NASequence.pxd

```diff
@@ -0,0 +1,117 @@
+from libcpp cimport bool
+from Types cimport *
+from String cimport *
+from Residue cimport *
+from EmpiricalFormula cimport *
+from Ribonucleotide cimport *
+from Map cimport *
+
+cdef extern from "<OpenMS/CHEMISTRY/NASequence.h>" namespace "OpenMS":
+
+    ctypedef Ribonucleotide RibonucleotideChainEnd
+
+    cdef cppclass NASequence "OpenMS::NASequence":
+        # wrap-hash:
+        #   toString().c_str()
+        #
+        # wrap-doc:
+        #   Representation of an RNA sequence
+        #   This class represents nucleic acid sequences in OpenMS. An NASequence
+        #   instance primarily contains a sequence of ribonucleotides. 
+
+        NASequence() nogil except +
+        NASequence(NASequence) nogil except + # wrap-ignore
+
+        bool operator==(const NASequence & rhs) nogil except +
+        bool operator!=(const NASequence & rhs) nogil except +
+        bool operator<(const NASequence & rhs) nogil except +
+
+        libcpp_vector[ const Ribonucleotide * ] getSequence() nogil except +
+
+        #Ribonucleotide * operator[](size_t index) nogil except +
+
+
+
+        # check if sequence is empty
+        bool empty() nogil except +
+
+        void setSequence(const libcpp_vector[ const Ribonucleotide *] & seq) nogil except +
+
+        # returns the peptide as string with modifications embedded in brackets
+        String toString() nogil except +
+
+        # sets the 5' modification
+        void setFivePrimeMod(const RibonucleotideChainEnd * modification) nogil except +
+
+        # returns the name (ID) of the N-terminal modification, or an empty string if none is set
+        const RibonucleotideChainEnd * getFivePrimeMod() nogil except +
+
+        # sets the 3' modification
+        void setThreePrimeMod(const RibonucleotideChainEnd * modification) nogil except +
+
+        const RibonucleotideChainEnd * getThreePrimeMod() nogil except +
+
+        # returns the residue at position index
+        #Ribonucleotide * get(Size index) nogil except +
```

Comment:
@poshul can you actually do anything with this Python object? If you cannot access the individual nucleotides, it seems to me its difficult to use, right?